src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -77,63 +78,88 @@ def f2008_parser():
 end module dummy_mod
 '''
 
+# Class Fparser2Reader
```

Comment:
Same. The double line-break before the comments would look more natural.

Reply:
Modified.